docker/README.md

```diff
@@ -61,15 +61,17 @@ documentation for further help with creating these two OAuth applications.
 The configuration files to be updated are:
 
 1. docker/.env
-   please see [docker README](../deploy/docker/SERVER.md) for help
+   please see [docker README](../deploy/docker/COMPOSE.md) for help
    with updating this config file)
 1. docker/conf.dev
    please see [server docs](../docs/admin/servers/auth.md) for help
    with updating this config file)
-1. client/config/local.js
+1. client/config/dev.js
```

Comment:
It was my mistake. This file is `local.js`; no need to update here. @nichlaes please confirm this.

Reply:
@prasadtalasila  On top of mind I must admit I'm not sure. If I need to look into it, I can do that monday.

Reply:
I guess that it does not matter, the updates above to dev.js are the same except for the environment name. Do we then need both?
We changed it from local.js to dev.js to help make development easier, so I would assume that keeping it a dev.js makes more sense semantically.

Reply:
A correct version of README.md has already been merged in your previous PR. Please remove this and other markdown files from this PR.